test/dotcom_web/live/trip_planner_test.exs

```diff
@@ -96,9 +126,7 @@ defmodule DotcomWeb.Live.TripPlannerTest do
         }
       }
 
-      expect(OpenTripPlannerClient.Mock, :plan, fn _ ->
-        {:ok, %OpenTripPlannerClient.Plan{itineraries: []}}
-      end)
+      stub_otp_results([])
```

Comment:
`stub_otp_results` is called in two places - once here and once in `stub_populated_otp_results`. I guess I could inline it in both places, but I wanted to avoid duplicating 👇

```elixir
expect(OpenTripPlannerClient.Mock, :plan, fn _ ->
  {:ok, %OpenTripPlannerClient.Plan{itineraries: itineraries}}
end)
```